lib/detect-libc.js

```diff
@@ -39,13 +43,31 @@ const safeCommandSync = () => {
  */
 const GLIBC = 'glibc';
 
+/**
+ * A Regexp constant to get the GLIBC Version.
+ * @type {string}
+ */
+const RE_GLIBC_VERSION = /GLIBC\s(\d+\.\d+)/;
+
 /**
  * A String constant containing the value `musl`.
  * @type {string}
  * @public
  */
 const MUSL = 'musl';
 
+/**
+ * This string is used to find if the {@link LDD_PATH} is GLIBC
+ * @type {string}
+ */
+const GLIBC_ON_LDD = GLIBC.toUpperCase();
+
+/**
+ * This string is used to find if the {@link LDD_PATH} is musl
+ * @type {string}
+ */
+const MUSL_ON_LDD = MUSL.toLowerCase();
```

Comment:
I did this just to make sure the text is always lowercase and a change to musl won't affect this constant.
But better than removing `.toLowerCase` could be just putting the string literal, since they are different information even though it is the same string.

Reply:
I created https://github.com/lovell/detect-libc/pull/20